invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -151,6 +151,7 @@ const LogoUploader = ({ community, defaultLogo, hasLogo, onError }) => {
               fluid
               wrapped
               rounded
+              className="tall community-logo"
```

Comment:
I find this class name misleading. How do you know the image will be tall ?

Reply:
`tall` is a relative term of course, How i would interpret it is that this logo will be taller than all other community logos in the application, but you are right that, as this is only used on one page (and probably will stay like that) i could make it specific to the settings page.